components/channel_notifications_modal/index.js

```diff
@@ -0,0 +1,26 @@
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import ChannelNotificationsModal from './channel_notifications_modal.jsx';
+
+import {NotificationLevels} from 'utils/constants.jsx';
+
+import {updateChannelNotifyProps} from 'actions/channel_actions.jsx';
```

Comment:
If you look at this action inside of `actions/channel_actions.jsx`, you'll notice it's really just a wrapper around a similarly named function from `mattermost-redux/actions/channels`. So instead of importing this from `actions/channel_actions.jsx`, we can actually get it from here `mattermost-redux/actions/channels`

We can then probably delete the wrapper in `actions/channel_actions.jsx`

Reply:
@jwilander , i think this wrapper is needed as have kept all the async calls related to channels inside actions/channel_actions.jsx . It would be more maintainable.

Reply:
@jwilander , please respond on this

Reply:
Sorry for the late response.

Not sure why it's needed or more maintainable, having the wrapper is practically just extra code for no benefit that I can see. The only reason it exists at all is because using wrappers was easier for migrating to redux actions in the first place

Let me know if I'm missing something but I'm not sure I am